lib/node_modules/@stdlib/array/base/take-map/docs/repl.txt

```diff
@@ -0,0 +1,91 @@
+
+{{alias}}( x, indices, mode, callback )
+    Takes elements with given indices from x and 
+    pass the values from tha call back function.
+
+    If `indices` is an empty array, the function 
+    returns an empty array.
```

Comment:
Desired line width in `repl.txt` is 80 characters for good terminal display without line wrapping, but we do want to use as much space as possible:

```suggestion
    Takes elements from an array and returns a new array after applying a
    mapping function.

    If `indices` is an empty array, the function returns an empty array.
```